src/fields/Fields.H

```diff
@@ -537,4 +490,21 @@ private:
     std::string m_insitu_file_prefix = "diags/field_insitu";
 };
 
+struct CheckDomainBounds {
```

Comment:
A one-liner explanation would help. Is this correct?
```suggestion
/** Helper struct to check whether a point is within a valid domain. */
struct CheckDomainBounds {
```